ui/job-view/JobView.jsx

```diff
@@ -183,16 +203,18 @@ class JobView extends React.Component {
       getWindowHeight() * (1 - latestSplitPct / 100);
 
     return (
-      <React.Fragment>
+      <div className="d-flex flex-column h-100">
+        <Favicon url={thFavicons[currentTreeStatus] || thFavicons.open} />
```

Comment:
It feels a bit strange to have the favicon be defined in the `<body>` markup. I also wonder whether having no non-JS fallback will mean the favicon will (a) be blank until the page is fully loaded, (b) not be picked up in some cases (eg browser bookmarks etc).

Updating an existing `<head>` favicon via native JS can be as short as:

```js
document.getElementById('favicon').href = '<new URL>';
```

Given the above, and that we don't use any of [react-favicon](https://github.com/oflisback/react-favicon)'s more advanced features (animations, dynamic bubbles, ...) how do you think the native JS approach might compare?

Reply:
You can also use refs to access the dom node directly :)

Reply:
I'm not sure I follow how that would work? :-)

Reply:
er, my bad. I didn't see the part where the link element with id 'favicon' is in the index.html file.... :)